docusaurus/docs/welcome/instructions/app-icon.md

```diff
@@ -0,0 +1,192 @@
+---
+id: app-icon
+title: App icon
+sidebar_label: App icon
+---
+
+Every app should have an impactul icon that stands out from all the others apps available on the user's device.
+
+An app icon is a graphic that represents your app to users. It can:
+
+-   Appear in the list of apps installed on a device and on the Home screen.
+-   Represent shortcuts into your app (for example, a contact shortcut icon that opens detail information for a contact).
+-   Help users find your app on Google Play and App Store.
+
+> ℹ️ An app icon is mandatory to publish the app to TestFlight/App Store and Google Play
+
+### How to generate the assets
+
+First and foremost, we need to take into consideration that automatic generation won't work everytime.
+Sometimes, small adjustments to an icon may be necessary and that should probably be done manually so it looks exactly how it is supposed to.
+There are design tools that solve this problem, e.g. Sketch and Adobe Illustrator, and they can provide automatic exportation under the Designer's control and customization. Nonetheless, automatic generation should always be the first step.
```

Comment:
Rephrased it a bit, but I think it looks much better already!

```suggestion
There are design tools that solve this problem, e.g. Sketch and Adobe Illustrator, which allow further customization and can assist designers in automatically exporting the icons. Nonetheless, automatic generation from a single asset should always be the first step to take as it is the most efficient and effective for most use cases.
```

Reply:
This looks better indeed, done 👍